getconfig/getconfig.sh

```diff
@@ -13,26 +13,27 @@ fi
 
 echo "Updating PHP configuration"
 DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null && pwd )"
-scp "$DIR"/getconfig.php "${HOST}":/tmp/
+scp -i /Users/davidsumner/.ssh/id_rsa_4096 "$DIR"/getconfig.php ridata.prod@"${HOST}":/tmp/
+echo "Ping"
 for PHP in "${SUPPORTED_PHP[@]}"; do
     echo "Getting config for $HOST -> $PHP"
-    ssh "$HOST" "/usr/local/php${PHP}/bin/php" /tmp/getconfig.php acquia > php/"${PHP}"-config
+    ssh -i /Users/davidsumner/.ssh/id_rsa_4096 -l ridata.prod "$HOST" "/usr/local/php${PHP}/bin/php" /tmp/getconfig.php acquia > php/"${PHP}"-config
```

Comment:
Looks like some local paths in here - perhaps add an env var or arg to set these locally when needed (could potentially use .ssh/config for some of this?)

Reply:
The arguments weren't used in the original copies of the script.  I just added them to my local copies.  Updating the .ssh/config file is the better way to go.  I will remove them from here.